src/main/java/org/zalando/logbook/Obfuscator.java

```diff
@@ -24,6 +24,8 @@
 import java.util.function.BiPredicate;
 import java.util.function.Predicate;
 
+// TODO implement
+// TODO find a nice way for body obfuscation
 @FunctionalInterface
 public interface Obfuscator {
```

Comment:
The api with the check for `(!Objects.equals(replacement, value))` looks a bit awkward. Is there a usecase where the obfuscator needs to decide on the value? I would prefer a separate method for checking whether an obfuscator applies to a header name.